### PR TITLE
Convert to actions

### PR DESCRIPTION
**Pull request by jason-jackson**
### Changed
- Convert to actions
- Update dependencies

### CODE REVIEW

**Review by briskt — APPROVED** [review]
Approved, but it's a bit redundant. Do you want approval from @forevermatt also?

**Comment by jason-jackson** [comment]
> Approved, but it's a bit redundant. Do you want approval from @forevermatt also?

Ideally, as it has dependency updates (that he did) that could possibly break things too.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ vars.DOCKER_ORG }}/${{ github.event.repository.name }}
+          tags: |
+            type=ref,event=branch
```
Comment (forevermatt) [inline .github/workflows/ci.yml:28]: Do you want to add `,enable=${{ github.ref_name != 'main' }}` here? I think that would make this equivalent to what we had in Codeship (i.e. not pushing a 'main' tag).

**WARNING:** I have not verified the syntax on this, just copied and tweaked from line 30.

```suggestion
            type=ref,event=branch,enable=${{ github.ref_name != 'main' }}
```

**Review by forevermatt — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jason-jackson) [reply .github/workflows/ci.yml:28]: I'm fine with this.

**Pull request merged**